src/cli/index.js

```diff
@@ -57,6 +57,10 @@ commander.option(
   '--mutex <type>[:specifier]',
   'use a mutex to ensure only one yarn instance is executing',
 );
+commander.option(
+  '--no-emoji',
```

Comment:
Instead, we should have had `--emoji no` or `--emoji false`.
What if we explicitly want to _enable_ emojis on non-darwin platforms? 😕

Reply:
Why are you reviewing a PR merged almost two years ago?